Project/Assets/ML-Agents/Examples/Hallway/Scripts/HallwayAgent.cs

```diff
@@ -18,13 +18,15 @@ public class HallwayAgent : Agent
     Renderer m_GroundRenderer;
     HallwaySettings m_HallwaySettings;
     int m_Selection;
+    StatsRecorder statsRecorder;
 
     public override void Initialize()
     {
         m_HallwaySettings = FindObjectOfType<HallwaySettings>();
         m_AgentRb = GetComponent<Rigidbody>();
         m_GroundRenderer = ground.GetComponent<Renderer>();
         m_GroundMaterial = m_GroundRenderer.material;
+        statsRecorder = Academy.Instance.StatsRecorder;
```

Comment:
```suggestion
        m_StatsRecorder = Academy.Instance.StatsRecorder;
```
The Unity coding standard for says this should be prefixed with `m_` and `PascalCase`'d. There's no official public mirror, but [here's a copy](https://github.com/needle-mirror/com.unity.polybrush/blob/755cb3b982de374af4f2e2c1bc826689810cf1d1/CodeStyleGuide.cs~#L156-L157) someone scraped from a package: